src/pages/new/index.md

```diff
@@ -0,0 +1,62 @@
+---
+keywords:
+  - New UXP Features
+  - InDesign UXP Features
+---
+# What's New
+This section contains details about the latest UXP features available in InDesign.
+
+## UXP InDesign DOM
+InDesign DOM provides APIs to create and modify InDesign application content through UXP scripting. You can find more details about this at [InDesign API](https://developer.adobe.com/indesign/dom/api/). 
+>
+Prior to 18.4, the InDesign DOM was available in the global space of the UXP script by default.
+>
+Starting 18.4, InDesign DOM is now available as a Javascript module, and it can be retrieved on a need basis using require() API. 
+
+### Overview
+The following code retrieves the InDesign DOM from 18.4 release onwards. This object allows you to open documents, modify them, run menu items, and more using the latest DOM version supported by the application.
+
+```js
+let myInDesign = require("indesign");
```

Comment:
Have you made updates to the other existing recipes, based on this change?

Reply:
Done